build.sbt

```diff
@@ -58,7 +58,7 @@ lazy val scalaCheck = "org.scalacheck" %% "scalacheck" % "1.18.1" % Test
 lazy val publishSettings = Seq(
   publish / skip := false,
   licenses += License.Apache2,
-  publishTo := Some("GitHub Package Registry".at("https://maven.pkg.github.com/mblink/composefree")),
+  publishTo := Some("BondLink S3".at("s3://bondlink-maven-repo")),
```

Comment:
We need to add this to `project/plugins.sbt` to get the `s3://` URL to work

```scala
addSbtPlugin("com.frugalmechanic" % "fm-sbt-s3-resolver" % "0.23.0")
```

Reply:
Ah right, thanks. Updated.